### PR TITLE
Add ls changes to 2201.6.0 release note

### PR DESCRIPTION
**Pull request by mindula**
## Purpose
Add LS release note for the 2201.6.0 release.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

**Review by praneesha — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -88,8 +88,20 @@ To view bug fixes, see the GitHub milestone for Swan Lake 2201.6.0 of the reposi
...
+
+- Added completions for the `group by` clause.
+- Added inlay hint support for function call expressions and method call expressions to provide information about parameters.
+
 ### Improvements
 
+#### Language Server
+
+- Removed service template initialization from the lightweight mode.
+- Improved completions in the client resource access action node context.
```
Comment (malinthar) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:101]:
```suggestion
- Improved the completion support and signature help for client resource access actions.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -88,8 +88,20 @@ To view bug fixes, see the GitHub milestone for Swan Lake 2201.6.0 of the reposi
...
+- Improved the main function completion item.
+- improved completions in the named argument context.
+
 ### Bug fixes
```
Comment (malinthar) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:105]: Let's add a link to the LS bug fixes as well.

Eg: In the previous release note
image

**Review by malinthar — CHANGES_REQUESTED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md (hunk 2 above)
Comment (mindula) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:105]: There is a link already

**Review by malinthar — APPROVED** [review]
LGTM!

**Pull request merged**